docs/workflow/requirements/linux-requirements.md

```diff
@@ -1,65 +1,100 @@
+  * [Unsupported OSes](#unsupported-oses)
 
-General instructions for building are [here](../README.md).
-Instructions for building CoreCLR for Linux are [here](../building/coreclr/linux-instructions.md).
+This guide will walk you through the requirements to build _dotnet/runtime_ on Linux. Before building there is environment setup that needs to happen to pull in all the dependencies required by the build.
 
+There are two suggested ways to go about doing this. The first one is you are able to use the [provided Docker environments](https://github.com/dotnet/dotnet-buildtools-prereqs-docker) used in the official builds, or you can set up the environment yourself. The documentation will go over both ways. Using Docker allows you to leverage our existing images which already have an environment set up, while using your own environment grants you better flexibility on having other tools at hand you might need.
 
-Docker
-==================
-
-Install Docker; see https://docs.docker.com/install/.
+## Docker
 
+Install Docker. For further installation instructions, see [here](https://docs.docker.com/install/). You can find the official .NET images in [their Docker hub](https://hub.docker.com/_/microsoft-dotnet). You can also use the images used for the official builds. More details on these ones in the [Linux building instructions doc](/docs/workflow/building/coreclr/linux-instructions.md#docker-images).
 All the required build tools are included in the Docker images used to do the build, so no additional setup is required.
 
+## Environment
 
-Environment
-===========
-
-These instructions are written assuming the current Ubuntu LTS, since that's the distro the team uses. Pull Requests are welcome to address other environments as long as they don't break the ability to use Ubuntu LTS.
+These instructions are written assuming the current Ubuntu LTS, since that's the officially used distribution. Pull Requests are welcome to address other environments as long as they don't break the ability to use Ubuntu LTS.
 
 Minimum RAM required to build is 1GB. The build is known to fail on 512 MB VMs ([dotnet/runtime#4069](https://github.com/dotnet/runtime/issues/4069)).
 
-Toolchain Setup
----------------
+### Toolchain Setup
 
-Building the repo requires CMake 3.14.5 or newer on Linux. Add Kitware's APT feed to your configuration for a newer version of CMake. See their instructions at <https://apt.kitware.com/>. You may need to add LLVM's APT feed to your configuration to obtain the required version of clang/LLVM. See their instructions at <https://apt.llvm.org/>.
+Building the repo requires CMake 3.14.5 or newer on Linux. Add Kitware's APT feed to your configuration for a newer version of CMake (optional). See their instructions at their [website](https://apt.kitware.com/). You may also need to add LLVM's APT feed to your configuration to obtain the required version of Clang/LLVM (usually not needed). See their instructions [over here](https://apt.llvm.org/).
```

Comment:
"usually not needed" does not help somebody going through these instructions for the first time.

Are the custom feeds needed on current Ubuntu LTS? If not, we should simplify this and move the notes about adding custom feeds to a separate section below, and also explain when you need them.

Reply:
I don't see how "usually not needed" doesn't help. I've never had to do that. I only know it exists because it's in the docs but it would be confusing to imply something not commonly needed as a must for every time.

Reply:
The custom feeds were required in the past for older Ubuntu versions.

Reply:
The problem with "usually not needed" is that it is ambiguous - how do I know whether I need it? The instructions should have unambiguous path through them that lead to success.